continuous_integration/recipe/meta.yaml

```
@@ -11,40 +10,48 @@ source:
   git_url: ../..
 
 build:
-  noarch: python
   number: {{ GIT_DESCRIBE_NUMBER }}
-  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  skip: true   # [py2k]
+  entry_points:
+    - dask-sql-server = dask_sql.server.app:main
+    - dask-sql = dask_sql.cmd:main
+  string: py{{ python }}{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```

Sorry will need to get rid of the `.` as well

```suggestion
  string: py{{ python.replace(".", "") }}{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
```